master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -47,184 +41,163 @@ container to network AB.
->       ports:
->       - "100:200"
->       notTag: bananas
->       notNet: 10.3.0.0/16
->       notSelector: type=='apples'
->       notPorts:
->       - "105:110"
->   egress:
->   - action: allow
->     source:
->       selector: type=='application'
-> EOF
+cat << EOF | calicoctl apply -f -
+- apiVersion: v1
+  kind: profile
+  metadata:
+    name: databases
+    labels:
+      role: database
+- apiVersion: v1
+  kind: profile
+  metadata:
+    name: apps
+    labels:
+      role: app
+EOF
 ```
 
-#### View the policy associated with the network
+Here the profiles contain no rules, merely labels that will be applied to each endpoint
+in the corresponding network.  So, for example, containers using the `databases` network
+will have the label `role = database` applied to it.
+
+#### 3. Create policy
+
+Policy resources are defined globally.  The selector in the profile selects which
+endpoints the policy applies to based on the endpoint labels.  Each policy contains
+a set of ingress and egress rules and actions where each rule can filter packets 
+based on a variety of source or destination attributes including selector based filtering.
 
-You can use the `calicoctl get -o yaml profile <profile>` to display the
-rules in the profile associated with the `databases` network.
+For our use case, we create a policy resource that applies to database endpoints
+and only allows ingress TCP traffic from app endpoints to destination port 3306.
+We also create policy to allow egress traffic from the apps.
+
+We can use `calicoctl create` to create two new policies for this:
 
 ```
-$ calicoctl get profile databases -o yaml
+cat << EOF | calicoctl create -f -
 - apiVersion: v1
-  kind: profile
+  kind: policy
   metadata:
-    labels:
-      foo: bar
     name: databases
   spec:
-    egress:
-    - action: allow
-      destination: {}
-      source:
-        selector: type=='application'
+    order: 0
+    selector: role == 'database'
     ingress:
-    - action: deny
-      destination:
-        net: 10.2.0.0/16
-        notNet: 10.3.0.0/16
-        notPorts:
-        - 105:110
-        notSelector: type=='apples'
-        notTag: bananas
-        ports:
-        - 100:200
-        selector: type=='application'
-        tag: alphatag
-      icmp:
-        code: 6
-        type: 10
-      notICMP:
-        code: 255
-        type: 19
-      notProtocol: udp
+    - action: allow
       protocol: tcp
       source:
-        net: 10.0.0.0/16
-        notNet: 10.1.0.0/16
-        notPorts:
-        - 1050
-        notSelector: type=='database'
-        notTag: bartag
+        selector: role == 'app'
+      destination:
         ports:
-        - 1234
-        - "10:20"
-        selector: type=='application'
-        tag: production
-    tags:
-    - tag1
-    - tag2s
+        -  3306
+- apiVersion: v1
+  kind: policy
+  metadata:
+    name: apps
+  spec:
+    order: 0
+    selector: role == 'app'
+    egress:
+    - action: allow
```

Comment:
For completeness, can we only allow traffic destined to the designated port of the database?